src/compute/src/render/join/delta_join.rs

```diff
@@ -364,6 +370,10 @@ where
             |_timer, count| count > 1_000_000,
             // TODO(mcsherry): consider `RefOrMut` in `half_join` interface to allow re-use.
             move |key, stream_row, lookup_row, initial, time, diff1, diff2| {
+                // Check the shutdown token to avoid doing unnecessary work when the dataflow is
```

Comment:
Is there any benefit to also checking the token in the yield function?  Might save a bunch of sorting and buffer manipulation.

Reply:
Oh yeah, for some reason I was thinking about that from the point of view of if that dataflow would eventually get dropped, which will not happen, so yielding on shutdown would be bad.

Reply:
Right, unfortunately with this approach we still need to grind through all the join matches produced by input data that has made it to the operator so far. There is really no way around this, short of modifying the implementation in DD to do a token check as well. Or `drop_dataflow` of course :)